electron/renderer/src/components/Sidebar.jsx

```diff
@@ -101,7 +105,7 @@ export default connect(
     currentAccentID: (accounts.find(account => account.visible) || {}).accentID,
     hasCreatedAccount: accounts.some(account => account.userID !== undefined),
     hasReachedLimitOfAccounts: accounts.length >= 3,
-    isAddingAccount: accounts.length && accounts.some(account => account.userID === undefined),
+    isAddingAccount: accounts.length > 0 && accounts.some(account => account.userID === undefined),
```

Comment:
Actual fix

Reply:
What's the change? A truthy check on length is the same as length > 0. No need for the extra code